dolphinscheduler-standalone-server/src/main/resources/application.yaml

```diff
@@ -90,7 +90,7 @@ registry:
 
 security:
   authentication:
-    # Authentication types (supported types: PASSWORD,LDAP)
+    # Authentication types (supported types: PASSWORD,LDAP,CASDOOR_SSO)
```

Comment:
The `dolphinscheduler-api`  application.yaml should update too.

Reply:
Done